test/parallel/test-buffer-slow.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const buffer = require('buffer');
+const Buffer = buffer.Buffer;
+const SlowBuffer = buffer.SlowBuffer;
+
+const ones = [1, 1, 1, 1];
+
+// should create a Buffer
+let sb = new SlowBuffer(4);
+assert(sb instanceof Buffer);
+assert.strictEqual(sb.length, 4);
+sb.fill(1);
+assert.deepEqual(sb, ones);
+
+// underlying ArrayBuffer should have the same length
+assert.strictEqual(sb.buffer.byteLength, 4);
+
+// should work without new
+sb = SlowBuffer(4);
+assert(sb instanceof Buffer);
+assert.strictEqual(sb.length, 4);
+sb.fill(1);
+assert.deepEqual(sb, ones);
+
+// should work with edge cases
+assert.strictEqual(SlowBuffer(0).length, 0);
+assert.strictEqual(SlowBuffer(buffer.kMaxLength).length, buffer.kMaxLength);
```

Comment:
This is a 1GB buffer on ia32 and 2GB on x64, and has a higher probability to fail on allocation. Especially for small devices like the rasp1. I'd recommend doing something like:
```js
try {
  assert.strictEqual(SlowBuffer(buffer.kMaxLength).length, buffer.kMaxLength);
} catch(e) {
  assert.equal(e.message, 'Buffer allocation failed - process out of memory');
}
```